x-pack/plugins/index_management/public/application/sections/home/data_stream_list/data_stream_list.tsx

```diff
@@ -166,16 +170,16 @@ export const DataStreamList: React.FunctionComponent<RouteComponentProps<MatchPa
 
   if (isLoading) {
     content = (
-      <SectionLoading>
+      <PageLoading>
```

Comment:
Same here. It seems we did some copy/paste of this pattern 😊

Reply:
I'm not sure what you're suggesting here, can you clarify? If it's the same comment as https://github.com/elastic/kibana/pull/101548#discussion_r656043185 then I'll defer to this later when we rearchitect for a consistent pattern across apps.

Reply:
Yes it was the same comment about early return. It's OK to leave it as is, goes down to preferences. 👍